git_machete/tests/functional/test_machete.py

```diff
@@ -2836,13 +2840,3 @@ def test_clean(self) -> None:
             """
         )
         self.assert_command(['status'], expected_status_output)
-
-def suite():
-    suite = unittest.TestSuite()
-    suite.addTest(MacheteTester('test_clean'))
-    return suite
-
-
-if __name__ == '__main__':
```

Comment:
😯 I've never scrolled that far, didn't realize it was here 😅

Reply:
Doesn't like like a proper way to execute tests indeed 😛

Reply:
yeah, i just use it to debug the tests